app/components/SignUp/CustomerInfo/CustomerInfoForm.js

```diff
@@ -31,8 +38,12 @@ export class CustomerInfoForm extends React.Component {
           <h2>{t('account:customer.header.info')}</h2>
           <FormSection name="customerInfo">
             <CustomerType handleChange={this.handleChange} />
-            <Product customerType={customerType} />
-            <ChannelType productType={productType} clearChannelType={this.handleCleanChannel} />
+            <Product customerType={customerType} clearChannelType={this.handleCleanChannel} />
+            {
+              productType === 'managed_vmc'
+                ? <ChannelType channelType={this.props.channelType} handleChannelUpdate={this.handleChannelUpdate} />
+                : null
```

Comment:
you may not need to return a null value here. 
```suggestion
              productType === 'managed_vmc' && <ChannelType channelType={this.props.channelType} handleChannelUpdate={this.handleChannelUpdate} />
```